src/TwoBucks/UpdateProfile.java

```diff
@@ -1,75 +1,96 @@
+/**
+ * Update Profile Feature US.24
+ *
+ * The UpdateProfile class is used when the user chooses to update
+ * their old first name, last name, or email to a new one. It will
+ * follow the pipe and filter architecture style because it uses data retrieved
+ * from another portions of the program, edits it, and outputs it in
+ * its original format
+ *
+ * @author Frino Jais
+ * @version 1.0
+ * @since 03-04-2020
+ */
+
 import java.util.*;
 import java.io.*;
 
 public class UpdateProfile {
-
-    public static void readTxtFile(String[] args) throws Exception{
+    /***
+     * This function will read from the text file where the
+     * User information is stored and change the user's choice
+     * of first name, last name, or email address
+     *
+     * @param args
+     * @throws Exception
+     */
+    public static void readFile(String[] args) throws Exception{
+        // creating object for User.txt where data is saved
         File txtFile = new File("User.txt");
```

Comment:
Save User already handles saving changes to .txt file

Reply:
Done